mephisto/operations/operator.py

```diff
@@ -595,4 +606,6 @@ def wait_for_runs_then_shutdown(
                 exc_info=False,
             )
         finally:
+            if assignment_id_file_path is not None:
+                os.remove(assignment_id_file_path)
```

Comment:
Delete the file with the assignmentId when the server is shut down as this file should be temporary to prevent confusion.